app/Services/Campaign/SubmissionService.php

```diff
@@ -66,7 +66,7 @@ public function process(array $data): string
                         'reason' => $rejection
                     ]));
         } else {
-            $this->approve((int) Arr::get($data, 'role_id'));
+            $this->approve((int) Arr::get($data, 'role_id'), $data);
```

Comment:
This code looks weird, the first Param is getting the role is from the data, but the second Param is the whole data. Can we either only use data, or call the method with the two params we need?